default.conf.js

```diff
@@ -5,7 +5,6 @@ const ProxySettings = require('./src/integration-test/config/proxy-settings').Pr
 
 exports.config = {
   name: 'citizen-integration-tests',
-  bootstrap: './src/integration-test/bootstrap/bootstrap.ts',
```

Comment:
There must be easier way of doing it. I started looking into `bootstrapAll` which is executed only once per run regardless how many workers are there.